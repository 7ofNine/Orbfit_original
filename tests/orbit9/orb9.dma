   1 588      
t0=   2.4574005000000000D+06 ;  from catalogue
coosys='EQU HEL INVL1B RAD';ilce=   1
  a(au)      h      k      p      q     lambda  nrev  gam                                           
  0.0000000000000000D+00
  0.5205906152079234D+01  0.1467892823238214D+00  0.8547891031047836D-02 -0.7940145612678347D-01  0.6504872332532352D-01  0.3931687376983306D+01         0   0.9818708766343381D-01        0  0.0000E+00
   2.5386938313878686       0.42551871390901952        2.5770398616061816     
   4.2779225057005134E-002   1.1682555169603450        1.5293072335686051     
  0.2500000000000000D+05
  0.5231092454756975D+01 -0.3020689670591119D-01  0.4321549097479083D-01 -0.7639659203529095D-01 -0.6029710567259938D-01  0.9180949236741658D+00      2108   0.2040878655122015D+01      169  3.2613E+00
  -72.555680354407272        55.538188255038300       -32.558968342696168     
  -24.481952009308230       -42.357642337733182        2.5209028382764167     
  0.5000000000000000D+05
  0.5221206371072484D+01  0.2893891336448342D-02 -0.1248638040608203D+00  0.3878660011672995D-01 -0.8823620175824214D-01  0.4627544136926113D+01      4215   0.4802756259881650D+00      338  3.1720E+00
   82.430663588542458        12.168596936072209       -24.900401633180966     
  -3.3232600881339316        41.365153615128726       -19.557319478282452     
  0.7500000000000000D+05
  0.5224473226035069D+01  0.8708096558402083D-01 -0.1596270208423769D-01  0.1006614266311896D+00  0.1672885835826577D-01  0.1606953311331474D+01      6323   0.2355354163682299D+01      507  3.1913E+00
  -68.554031258239903       -4.1979958279617087        48.568236483762831     
  -3.0508816098868556       -48.812107371569752       -6.5876092475593051     
  0.1000000000000000D+06
  0.5234765496058384D+01 -0.9392695941870775D-01  0.1613376094665776D-01  0.3447671571304544D-02  0.1013878737511526D+00  0.5230838693908746D+01      8430   0.1500649929505956D+01      676  3.5921E+00
   95.715137513550729        55.126547559398780        35.088910156184021     
  -39.952805991043427        56.457897361144155        37.213742802046447     
