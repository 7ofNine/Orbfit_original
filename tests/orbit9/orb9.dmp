   4 JUPITER   SATURN    URANUS    NEPTUNE  
t0=   2.4574005000000000D+06 ;  JPL DE 431 13 GEN 2016 0.0000
coosys='EQU HEL INVL1B RAD';ilce=   0
  a(au)      h      k      p      q     lambda  nrev  gam                                           
; planetary inverse masses as used
  9.9999402303776952D-01  1.0473486309727623D+03  3.4979017677866345D+03
  2.2902950523706935D+04  1.9412259775973071D+04
  0.0000000000000000D+00
  0.5202129950115880D+01  0.9122242083634373D-02  0.4805013544754314D-01 -0.2072393467639870D-02  0.1964699376174683D-02  0.2746740698223678D+01         0
  0.9558194405534699D+01  0.5320511726855959D-01 -0.3972385092655666D-03  0.7030411241100789D-02 -0.4091599084335329D-02  0.4228712516713403D+01         0
  0.1914221308704462D+02  0.1029256672648416D-01 -0.4863655323779833D-01 -0.7016377605402102D-02  0.5606181307608352D-02  0.3225887484065556D+00         0
  0.2997467156632188D+02  0.6595277430755391D-02  0.3512833921159024D-02 -0.1223467034155880D-02 -0.6235033008700971D-02  0.5869559187795051D+01         0
  0.2500000000000000D+05
  0.5202877442126920D+01  0.4767143398959212D-01  0.1112052502964836D-01  0.3328664111688499D-02 -0.1220927092772934D-02  0.6205592514220355D+01      2107
  0.9524898831597124D+01 -0.3076357802373415D-02  0.5486767691647607D-01 -0.6303986235524022D-02  0.4370690170878608D-02  0.2315267033773367D+01       849
  0.1916617273141340D+02 -0.8165653538914330D-02 -0.4088486027105363D-01 -0.8053257691527144D-02  0.2550730214956886D-02  0.3742213966955964D+01       297
  0.3003889244389588D+02  0.7728192829041814D-02  0.7447843691164253D-02 -0.5115663808545385D-03 -0.6073535783246731D-02  0.4144662197350261D+01       152
  0.5000000000000000D+05
  0.5202933972865300D+01  0.3810653890431208D-01  0.1526304961751467D-01 -0.2203284225727046D-02  0.2029692338211530D-02  0.3454315709466498D+01      4215
  0.9519890184632539D+01  0.5320572604379051D-01 -0.4557210887459156D-01  0.7467915288251287D-02 -0.3143523911518883D-02  0.2213885128667659D+00      1698
  0.1917059702044229D+02 -0.2629169859867949D-01 -0.2804885033107653D-01 -0.9310639215320909D-02 -0.2762001611877075D-03  0.8940436447615226D+00       595
  0.3015155147597498D+02  0.1028987679672773D-01  0.1721856877635920D-02 -0.2561226981317625D-04 -0.5749231581335393D-02  0.2408362080497216D+01       304
  0.7500000000000000D+05
  0.5202480562270028D+01  0.4274208307370289D-01 -0.3609564286782917D-01  0.3381234144280252D-02 -0.5880514850724045D-03  0.6386480824852985D+00      6323
  0.9532958113465993D+01  0.2325504308004874D-01  0.3031767516301787D-01 -0.6716222050735229D-02  0.3968982279481680D-02  0.4571390572603942D+01      2546
  0.1917198039868353D+02 -0.3188176769724304D-01 -0.1067366657366316D-01 -0.8149237203790681D-02 -0.3689172961908210D-02  0.4318594494332964D+01       892
  0.3022407864222144D+02  0.1365125972522450D-01  0.7400015868005893D-02  0.4884450811287783D-03 -0.5469020567724756D-02  0.6715150372374771D+00       456
  0.1000000000000000D+06
  0.5201743989244517D+01  0.2802392503183667D-01 -0.2120737650593308D-01 -0.2526144943941683D-02  0.2050539655834421D-02  0.4175304288656225D+01      8430
  0.9553997115623455D+01  0.8331101368632439D-02 -0.7991607906030876D-01  0.7558500005287665D-02 -0.2109223473469847D-02  0.2465761236029849D+01      3395
  0.1915416974305473D+02 -0.3269877857428152D-01  0.6443581270538581D-02 -0.7131786673221466D-02 -0.6200493411598483D-02  0.1452643131821723D+01      1190
  0.3014929864388791D+02  0.8627740936701597D-02  0.9518182106860139D-03  0.7074104338560309D-03 -0.5152117005883305D-02  0.5225789347031484D+01       607
