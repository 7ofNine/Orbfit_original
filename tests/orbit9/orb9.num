 hmax=   0.10000 iord= 12 epms=  1.0000D-12iork= 12 eprk=  1.0000D-10
  10   4   1  10   0
 npas    10 ep    0.106D-10
