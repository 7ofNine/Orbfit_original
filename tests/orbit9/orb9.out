; orbit9 vers. c, output file
